src/apps/Agreement/ConfigurationChecklist.js

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Box,
+  useLayout,
+  GU,
+  useTheme,
+  ProgressBar,
+  textStyle,
+  Button,
+  springs,
+} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import checklistCompleteGraphic from './assets/checklist-complete.png'
+import { Transition, animated } from 'react-spring'
+
+function countProgress(items) {
+  return items.filter(([_, checked]) => checked).length
+}
+
+const AnimatedDiv = animated.div
+
+function ConfigurationChecklist({ items, onClose }) {
+  const [progress, setProgress] = useState(countProgress(items))
+  const theme = useTheme()
+
+  const barProgress = (1 / items.length) * progress
+  const checklistComplete = items.length === progress
+
+  useEffect(() => {
+    setProgress(countProgress(items))
+  }, [items])
```

Comment:
Not sure if we plan on expanding the `useEffect()` in the future, but as it is, it would be simpler if we used a `useMemo()` for `countProgress`.